iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi_process.hpp

```diff
@@ -156,6 +160,8 @@ class ProcessManager : public ProcessManagerInterface
 
     void addConditionVariableForProcess(const ProcessName_t& processName) noexcept;
 
+    void addEventVariableForProcess(const ProcessName_t& processName) noexcept;
```

Comment:
Do you want me to remove the whole event variable integration. This would also mean that we again refactor the simple example we have written, we rewrite the ActiveCallSet and the tests?!